CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## unreleased
 
+* GooglePay
+  * Fixing a crash on API 33 devices. It is recommended that merchants not use 5.1.0 for GooglePay.
```

Comment:
Is this issue specific to 5.1.0 or would it also be 5.0.0?

Reply:
Looks like it was introduced in 5.1.0. See the diff 5.0.0...5.1.0 below for refs.

<img width="800" alt="Screenshot 2024-10-29 at 1 16 05 PM" src="https://github.com/user-attachments/assets/3d3f2d1a-d241-4d04-80cf-8cc140336b4a">